### PR TITLE
using granulate-utils DatabricksClient

### PR DESCRIPTION
**Pull request by roi-granulate**
## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the relevant documentation.
- [ ] I have added tests for new logic.

### CODE REVIEW

File: gprofiler/main.py [hunk 1]
```diff
@@ -35,7 +36,6 @@
 )
 from gprofiler.consts import CPU_PROFILING_MODE
 from gprofiler.containers_client import ContainerNamesClient
-from gprofiler.databricks_client import DatabricksClient
```
Comment (Jongy) [inline gprofiler/main.py:38]: Remove the old file please

Comment (roi-granulate) [reply gprofiler/main.py:38]: 64021b3

File: granulate-utils
Comment (Jongy) [inline granulate-utils:1]: This commit is the unmerged 2nd PR - we won't merge it as is. We can first merge the 2nd PR, the update the ref to granulate-utils updated master, OR update it now to the master after the 1st PR. I suggest we wait, so let's first merge the 2nd PR, https://github.com/Granulate/granulate-utils/pull/149

**Comment by Jongy** [comment]
This updates https://github.com/Granulate/granulate-utils/pull/149

**Pull request merged**